index.html

```diff
@@ -414,7 +414,7 @@ <h3>Multikey</h3>
 {
   "@context": [
     "https://www.w3.org/ns/did/v1",
-    "https://w3id.org/security/data-integrity/v1"
+    "https://w3id.org/security/data-integrity/v2"
```

Comment:
This example DID doc should actually be using the multikey context for the multikey terms in use below:

```suggestion
    "https://w3id.org/security/multikey/v1"
```